examples/colour-uniform/main.rs

```diff
@@ -1639,7 +1649,13 @@ impl<B: Backend> Drop for FramebufferState<B> {
                 device.destroy_fence(fence);
             }
 
-            for command_pool in self.command_pools.take().unwrap() {
+            for (mut command_pool, comamnd_buffer_list) in self.command_pools
```

Comment:
minor: this change isn't required as command pool destruction should implicitly free all associated command buffers if we follow vulkan semantics